### ISSUE TITLE
[Napoleon at Waterloo] InvalidPathException: Illegal char <:> at index 13: odds_marker_2:1.png

### ISSUE THREAD

**Issue by uckelman** [vassalengine/vassal#13313]
| ABR | |
|-----|-----|
| Version | 3.7.10 |
| Reporter | zovs66@gmail.com |
| Error Log | [13468 bytes](https://vassalengine.org/files/gh/44d3189419660c4e22810e1ce867d3a9a18b1308) |

<pre>After loading the tutorial and doing as instructed per the tutorial I opened the More Info from the Help menu and got this dialog instead.

Napoleon at Waterloo v1.0 3.7.10

java.nio.file.InvalidPathException: Illegal char <:> at index 13: odds_marker_2:1.png
 at java.base/sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:204)
 at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:175)
 at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
 at java.base/sun.nio.fs.WindowsPath.parse(WindowsPath.java:92)
</pre>

**Comment by BrentEaston** [comment]
Invalid counter name. Must have been created on a mac, but it breaking Windows.

**Comment by cholmcc** [comment]
> Invalid counter name. Must have been created on a mac, but it breaking Windows.

No, the image was created on Linux.  For some odd reason, Windows barks on the `:` in the file name - sigh!. 

It seems like this happens when the HTML viewer needs to get an image from the Zip archive.  Then, the image is extracted from the Zip archive and written to disk (as a temporary file).  However, windows makes a ton of noise over the `:`.  This should really be dealt with differently in the code - see #14290 

Yours,
_Christian_

**Comment by uckelman** [comment]
Fixed by #14323.
